_posts/en/newsletters/2024-11-01-newsletter.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'Bitcoin Optech Newsletter #327'
+permalink: /en/newsletters/2024/11/01/
+name: 2024-11-01-newsletter
+slug: 2024-11-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Timeout tree channel factories:** ZmnSCPxj [posted][zmnscpxj post1]
+  to Delving Bitcoin and [discussed][deepdive] with Optech contributors a
+  proposal for a new multi-layer [channel factory][topic channel
+  factories] design named _SuperScalar_.  A goal of the design is to
+  provide a construction that could be easily implemented by a single
+  vendor without waiting for large protocol changes that require
+  widespread agreement.  For example, a Lightning service provider (LSP)
+  that distributes wallet software could allow its users to more cheaply
+  open channels and receive inbound liquidity without sacrificing any of
+  LN's trustlessness.
+
+  The overall construction is based on a _timeout tree_, where a
+  _funding transaction_ pays to a tree of pre-defined child transactions
+  that are ultimately spent offchain into a large number of separate
+  payment channels.  After a configurable timeout (e.g. a month), some
+  of the parties involved in the timeout tree forfeit any of their funds
+  remaining in the tree---this incentivizes them to withdraw or find
+  alternative security for those funds before the expiry, which can
+  encourage the use of cheap offchain mechanisms rather than publishing
+  parts of the tree onchain.  In previously described timeout trees (see
+  [Newsletter #270][news270 timeout trees]), user funds that timed out
+  became the property of a service provider, but ZmnSCPxj reverses this
+  and makes a service provider's timed out funds become the property of
+  the users---this places the burden of getting transactions confirmed
+  on the service provider rather than end users.
+
+  The timeout trees used require each involved party to contribute a
+  signature.  This avoids the need for consensus changes but limits the
+  practical maximum number of users in a factory due to the well-known
+  [multi-signer coordination problem][news270 coordination].
+
+  Most of the leaves of the timeout tree are offchain funding
+  transactions for the common type of channel used today (LN-Penalty),
```

Comment:
It doesn't look like "LN-Penalty" has been defined anywhere in this repo/website other than with introductory phrases like here, or "In the current LN channel design" in a previous newsletter.

It is also indirectly referred to in the Eltoo topic: "Although eltoo can be used with a penalty mechanism similar to the one used with existing LN channels, eltoo doesn’t need the penalty mechanism in order to be secure."

For anyone not familiar with LN-Penalty, maybe a link to a resource or topic could be good for this term.